project/Settings.scala

```diff
@@ -0,0 +1,83 @@
+import sbt._
+
+/**
+ * Application settings and dependencies
+ */
+object Settings {
+  object Definitions {
+    /** The name of the application */
+    val name = "ocs3"
+
+    /** Project version */
+    val version = "2016001.1.1"
```

Comment:
@cquiroz if you do that the `sbt-release` plugin can roll the version for you (by rewriting that file).